crates/revm/src/state_change.rs

```diff
@@ -319,14 +319,14 @@ where
         let mut source_address = Address::ZERO;
         data.copy_to_slice(source_address.as_mut_slice());
 
-        let mut validator_public_key = FixedBytes::<48>::ZERO;
-        data.copy_to_slice(validator_public_key.as_mut_slice());
+        let mut validator_pubkey = FixedBytes::<48>::ZERO;
+        data.copy_to_slice(validator_pubkey.as_mut_slice());
```

Comment:
unnecessary to change variable name, pls revert

Reply:
In the next line, in `WithdrawalRequest`, `validator_public_key` changed to `validator_pubkey` in recent alloy. So if I revert this variable name, this means:

```rust
withdrawal_requests.push(Request::WithdrawalRequest(WithdrawalRequest {
            source_address,
            validator_pubkey: validator_public_key,
            amount,
        }));
```

More verbose and complex to read no? Do you still prefer this?

Reply:
ah ok I see, makes sense then